### PR TITLE
feat: add accordion component

### PR DESCRIPTION
**Pull request by afonsojramos**
The FAQ page was too much text imo. This should make it a bit more readable.

Also added the SSO bit of documentation mentioned in https://github.com/gitify-app/gitify/issues/747

### CODE REVIEW

File: src/components/solid/FaqAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  AccordionItem,
+  AccordionTrigger,
+} from '~/components/ui/Accordion';
+
+const content = [
+  {
+    id: '1',
+    question: "My notifications aren't showing?",
+    answer:
+      "Some organisations require applications to request access before allowing access to any data (including notifications) about their repositories.\n\nTo check if Gitify is approved by your organisation you can go to [settings](https://github.com/settings/applications), then click on **Gitify** and scroll to **Organization access**.\n\nAlternatively, if you generated a **PAT**, you can check if it has the correct permissions by going to [settings](https://github.com/settings/tokens) and checking if the token's SSO has been Authorized.",
```
Comment (setchy) [inline src/components/solid/FaqAccordion.tsx:14]: Do we need to reword this FAQ? Perhaps all it needs is a more specific header? 

I've reread it a dozen times and it seems to be covering 
s
- GitHub App or GitHub Oauth App approval for certain orgs (enterprise??)
- PAT when used with an org that has SSO/ SAML configured?

**Review by setchy — APPROVED** [review]

File: src/components/solid/FaqAccordion.tsx (hunk 1 above)
Comment (afonsojramos) [reply src/components/solid/FaqAccordion.tsx:14]: The SSO part was added in this PR 👀

Comment (setchy) [reply src/components/solid/FaqAccordion.tsx:14]: Yes I know... But I don't think regular GitHub Cloud users with PAT can configure that...

Comment (setchy) [reply src/components/solid/FaqAccordion.tsx:14]: Maybe to put it differently, if I'm a regular GitHub Cloud user, and "my notifications aren't showing" - these tips aren't applicable.

Comment (afonsojramos) [reply src/components/solid/FaqAccordion.tsx:14]: Sounds good! Alright, I'll merge this in and open another one in a few hours!

**Pull request merged**